src/Polly.Core/Utils/TimeSpanAttribute.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All Rights Reserved.
```

Comment:
Should this be an Apache 2.0 header (or whatever)?

Reply:
Sorry, this should not be here. Dropped.